src/components/Badge/Badge.tsx

```diff
@@ -26,8 +26,13 @@ class Badge extends React.Component<BadgeProps> {
     const { priority, children, className, icon } = this.props;
     return (
       <div
+        className={st(
+          classes.root,
+          { withIcon: !!icon },
+          classes[`priority-${priority}`],
+          className,
+        )}
```

Comment:
So, if I understand correct, what we're actually saying here, is that we want to forbid the use of "variant" states, and allow only booleans.
I wonder though, how can we enforce this?
Some kind of lint rule?

Reply:
I just think that we've using stylable wrong all along.
state are there to represents, well, logic states. styles variations are class variations and not states.

so I don't think we need a rule, but rather pass the knowledge and fix ourselves